specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/examples/SearchServiceCreateOrUpdateIndexer.json

```diff
@@ -35,7 +41,13 @@
           "maxFailedItemsPerBatch": 5
         },
         "fieldMappings": [],
-        "disabled": false
+        "disabled": false,
+        "encryptionKey": {
+          "keyVaultKeyName": "myUserManagedEncryptionKey-createdinAzureKeyVault",
+          "keyVaultKeyVersion": "myKeyVersion-32charAlphaNumericString",
+          "keyVaultUri": "https://myKeyVault.vault.azure.net",
+          "accessCredentials": null
```

Comment:
Are you really sending back `null` on the wire?

Reply:
yes, this is by design (and is what we did for other [resources](https://github.com/Azure/azure-rest-api-specs/blob/257f060be8b60d8468584682aa2d71b1faa5f82c/specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/examples/SearchServiceCreateOrUpdateIndex.json#L593)).

This is to let customers know that they have specified an explicit access credential and are not using the system assigned identity